src/wasm.cc

```diff
@@ -670,4 +670,28 @@ void clearWasmCachesForTesting() {
   }
 }
 
+namespace internal::testing {
```

Comment:
nit: for consistency with the existing `clearWasmCachesForTesting()` function (which immediately precedes these functions), can you either remove the `internal::testing` namespace and name these `*ForTesting()`, or else rename `clearWasmCachesForTesting()` to `clearWasmCaches()` and move it inside the namespace. (If you do the latter, I can take care of the trivial [adjustment](https://github.com/envoyproxy/envoy/blob/2311e03d91b146db83bac4c03f50aed4b7cf5e74/test/extensions/common/wasm/wasm_test.cc#L726) to Envoy code later.)

Reply:
Got it. I opted for renaming new functions to `*ForTesting()` in order to avoid breaking the existing interface.